app/services/albums.js

```diff
@@ -0,0 +1,11 @@
+const config = require('../../config'),
+  fetch = require('../../node_modules/node-fetch'),
```

Comment:
nope, dont reference node_modules directly. If you install a module just do `require('node-fetch')` . Also, generally we use `request` [https://github.com/request/request-promise](url)